ui/src/utils/api.ts

```diff
@@ -1,8 +1,9 @@
 // Libraries
 import {Client} from '@influxdata/influx'
 import {get} from 'lodash'
+import {getAPIBasepath} from 'src/utils/basepath'
 
-const basePath = '/api/v2'
+const basePath = getAPIBasepath() + '/api/v2'
```

Comment:
same nit here about template strings vs string concatenation

Reply:
do you just mean using ` `getAPIBasepath() + /api/v2` ` instead?